### PR TITLE
Add action list example

### PR DESCRIPTION
**Pull request by daileytj**
Fixes PXBLUE-1311.

#### Changes proposed in this Pull Request:
- Add action list design pattern example

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "cordova-plugin-splashscreen": "^5.0.2",
     "cordova-plugin-statusbar": "^2.4.2",
     "cordova-plugin-whitelist": "^1.3.3",
+    "i": "^0.3.6",
```
Comment (joebochill) [inline package.json:46]: did you do `yarn i @pxblue/angular-components`?

File: src/app/pages/list/action-list/action-list.component.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                <mat-icon>more_vert</mat-icon>
+            </button>
+            <mat-menu #menu="matMenu">
+                <button data-cy="remove" mat-menu-item (click)="onRemoveItem(i)">Remove</button>
+                <button data-cy="details" mat-menu-item>View Details</button>
+            </mat-menu>
+        </span>
+    </pxb-info-list-item>
+</mat-list>
+<div *ngIf="data.length === 0" style="position: absolute; top: 50%; left: 50%; transform: translate(-50%, -50%);">
```
Comment (joebochill) [inline src/app/pages/list/action-list/action-list.component.html:35]: can you classify these inline styles

**Review by joebochill — COMMENTED** [review]
I think it would behoove us to have a story on the backlog to update some of these patterns to be more 'mobile-y.' Like for this one, we should probably be opening the menu as a bottomsheet more like we do for the RN examples. 

However, for the sake of keeping these migrations fast, I am okay with porting stright from Angular as the starting point. But let's make that story and add some bullet points to the AC as we identify things during migration that should be updated.

File: package.json (hunk 1 above)
Comment (daileytj) [reply package.json:46]: i just might have accidentally... uggh. Ill clean that up

File: src/app/pages/list/action-list/action-list.component.html (hunk 2 above)
Comment (daileytj) [reply src/app/pages/list/action-list/action-list.component.html:35]: you got it

**Review by joebochill — APPROVED** [review]

**Pull request merged**